fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -97577,7 +97881,7 @@ creation_date: 2014-11-13T10:07:48Z
 id: FBbt:00110947
 name: larval serotonergic LP1-1 neuron
 def: "Larval serotonergic LP1 neuron whose cell body is located in the mid-lateral region, at the optic lobe margin. It projects medially to the basolateral region." [FlyBase:FBrf0219731]
-comment: Huser et al., (2012) describe two neurons with some variability in morphology. They assign both to the LP1-1 subtype.
+comment: Huser et al., (2012) describe two neurons with some variability in morphology. They assign both to the LP1-1 subtype. This is therefore equivalent to the LP1 class [FBC:CP].
```

Comment:
If you're confident that this is equivalent then worth merging.  Otherwise comment should state that the two terms are strong candidates for merging based on ...

Reply:
What was the procedure for this?

Reply:
Two options:

1. Make the losing stanza obsolete as follows, deleting all logical relationships first: 
![image](https://user-images.githubusercontent.com/112839/42315694-fd0e9442-803f-11e8-9aad-2b842bac91cf.png)
'term replaced by' => link back to winning term.  You can, and probably should, add information to the comment field about why the term was obsoleted.  You should also make the losing term a synonym of the winning unless this will be very confusing.

2. Do a (OBO) merge.  There's a rather exhaustive description of how to do this here: http://go-ontology.readthedocs.io/en/latest/MergeTerms.html.  

We used to do 2 a lot but I think we should switch to using 1 as it preserves IRIs and details of the original term.  I think this shouldn't need any prior announcement.  That should be reserved for cases where one term is obsoleted an multiple mappings are specified using 'consider'.